module/support/logging/MessageLogHandler/src/MessageLogHandler.cpp

```diff
@@ -0,0 +1,91 @@
+#include "MessageLogHandler.hpp"
+
+#include "extension/Configuration.hpp"
+
+#include "message/nuclear/LogMessage.hpp"
+#include "message/nuclear/ReactionStatistics.hpp"
+
+namespace module::support::logging {
+
+    using extension::Configuration;
+
+    namespace {
+        message::nuclear::ReactionStatistics to_message(const NUClear::message::ReactionStatistics& in) {
+
+            message::nuclear::ReactionStatistics out;
+
+            out.identifiers.name     = in.identifiers.name;
```

Comment:
This looks like it will require latest NUClear, which I have a PR for here: https://github.com/NUbots/NUbots/pull/1264

Reply:
Good catch. What's left for that to get merged?

Reply:
Just needs a review